src/kawaz/core/management/management/commands/start_livereload.py

```diff
@@ -30,7 +30,7 @@ def handle_noargs(self, **options):
         # STATICFILES_DIRS, TEMPLATE_DIRS
         for path in settings.STATICFILES_DIRS:
             pathlist.append("{}".format(path))
-        for path in settings.TEMPLATE_DIRS:
+        for path in settings.TEMPLATES[0]['DIRS']:
```

Comment:
`TEMPLATES` should be iterated over even though we currently have a single directory for the templates.